examples/basic_monitor/monitor.c

```diff
@@ -125,7 +125,7 @@ do_stats_display(struct rte_mbuf* pkt) {
         printf("Port : %d\n", pkt->port);
         printf("Size : %d\n", pkt->pkt_len);
         printf("Hash : %u\n", pkt->hash.rss);
-	printf("N°   : %d\n", pkt_process);
+	printf("N°   : %ld\n", pkt_process);
```

Comment:
@gkatsikas Shouldn't you be using `%ull` or `%" PRIu64 "` for printing uint64_t?  `%ld` is for a 32 bit signed ints where as we are dealing with 64 bit unsigned ints.